### PR TITLE
Updated release notes for Serverless TP 1.1.0

### PR DESCRIPTION
**Pull request by abrennan89**

### CODE REVIEW

**Comment by alanfx** [comment]
Looks good @abrennan89!

File: modules/serverless-rn-1-1-0.adoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+* {ServerlessProductName} has been updated to use Knative Serving 0.8.1.
+* Enhanced Operator metadata now includes more information regarding support state and a link to the official installation documentation.
+* A developer preview version of Knative Eventing is now available for use with {ServerlessProductName}, however this is not included in the {ServerlessProductName} Operator and is not currently supported as part of this Technology Preview. For more information, see https://openshift-knative.github.io/docs/docs/index.html[Knative Eventing on OpenShift Container Platform].
+
+[id="fixed-issues-1-1-0_{context}"]
+== Fixed issues
+
+* Users who were not project administrators would previously see the following error when using {ServerlessProductName}:
++
+```
```
Comment (bergerhoffer) [inline modules/serverless-rn-1-1-0.adoc]: Should this be "----"? This looks to be highlighting the text in some way.

File: modules/serverless-rn-1-1-0.adoc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+revisions.serving.knative.dev: User "sounds" cannot list resource "revisions
+```
++
+This issue has now been fixed with the addition of new RBAC rules.
+
+* A race condition was preventing Istio sidecar injection from working correctly. Istio did not consider the `knative-serving` namespace to be present in the ServiceMeshMemberRoll at the time of pod creation. Istio now waits for status information from ServiceMeshMemberRoll which fixes this issue.
```
Comment (bergerhoffer) [inline modules/serverless-rn-1-1-0.adoc]: Pod

File: modules/serverless-rn-1-1-0.adoc [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+* Users may see the error `Unknown Uninitialized : Waiting for VirtualService to be ready` while waiting for a service in a newly created namespace to be ready, which can take several minutes. If a user allows enough time between the creation of a namespace and the creation of a service in the namespace (approximately one minute), this error may be avoided.
+
+* Existing Knative Services and Routes which have been created with public visibility cannot be updated to cluster-local visibility. If you require cluster-local visibility on Knative Services and Routes, this must be configured at the time of creating these resources.
+
+* Knative service returns a 503 status code when the cluster is running for a
+long time. The Knative Serving Pods do not show any errors. Restarting the
+`istio-pilot` Pod temporarily fixes the issue.
+
+* The gRPC and HTTP2 do not work against routes. This is a known limitation of
```
Comment (bergerhoffer) [inline modules/serverless-rn-1-1-0.adoc]: Is this missing something after "HTTP2"? Or should "The" be removed, like "HTTP2 and gRPC do not work..."

**Comment by bergerhoffer** [comment]
@abrennan89 What versions is this for?

**Pull request merged**